internal/collector/collector_test.go

```diff
@@ -1,11 +1,18 @@
 package collector
 
 import (
+	"io/fs"
+	"strings"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 )
 
+type mockDirEntry struct {
+	name  string
+	isDir bool
+}
+
```

Comment:
Can we move this struct just above its methods? Or the other way around; just to be together.